pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
 dependencies:
   angular: 5.0.0-alpha+4
   angular_forms: 1.0.1-alpha+4
+  build: '>=0.11.1 <0.13.0'
```

Comment:
Just curious: so `build` isn't a dev dependency?

Reply:
In this case the package implements a custom `Builder` since it uses a non-standard extension for generated css files. There is a `lib/` import to `build` so it needs the dep.

Reply:
Got it, thanks!